### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by Ayagoumi**
- Removed unnecessary fonts
- Added back-to-top behavior when routing between pages
- Adjusted the suite footer responsiveness
- Fixed a typo in the font URLs imports
- Updated and added missing dependencies in package.json
- Added esModuleInterop to tsconfig.json
- Reverted the Webpack development configuration to use separate links (i.e  http://localhost:5002/remoteEntry.js for explorer and http://localhost:5003/remoteEntry.js for the wallet) and deleted the built files.

### CODE REVIEW

File: webpack.dev.js [hunk 1]
```diff
@@ -8,11 +8,11 @@ module.exports = merge(common, {
     devtool: 'inline-source-map',
 
     output: {
-        publicPath: 'http://localhost:3000/',
+        publicPath: 'http://localhost:5001/',
```
Comment (Chriserus) [inline webpack.dev.js]: Is this a necessary port change?

File: webpack.dev.js [hunk 2]
```diff
@@ -8,11 +8,11 @@ module.exports = merge(common, {
...
     },
 
     devServer: {
-        port: 3000,
+        port: 5001,
```
Comment (Chriserus) [inline webpack.dev.js]: Same here

File: webpack.dev.js [hunk 3]
```diff
@@ -22,8 +22,8 @@ module.exports = merge(common, {
             name: 'host_react',
             filename: 'remoteEntry.js',
             remotes: {
-                Explorer: 'Explorer@http://localhost:3000/explorer/remoteEntry.js',
-                wallet: 'wallet@http://localhost:3000/wallet/remoteEntry.js',
+                Explorer: 'Explorer@http://localhost:3001/remoteEntry.js',
+                wallet: 'wallet@http://localhost:3003/remoteEntry.js',
```
Comment (Chriserus) [inline webpack.dev.js]: And here. By default, these apps run on port 3000 and 3001 I think

Comment (Ayagoumi) [reply webpack.dev.js]: All ports have been changed for consistency and to be able to work with Kopernikus, without a change in Webpack dev environment.

**Review by Chriserus — DISMISSED** [review]

**Review by Chriserus — APPROVED** [review]

**Review by aeddaqqa — APPROVED** [review]
LGTM

**Pull request merged**